src/main/java/com/revlearn/team1/service/CourseService.java

```diff
@@ -0,0 +1,19 @@
+package com.revlearn.team1.service;
+
+import com.revlearn.team1.exceptions.CourseNotFoundException;
+import com.revlearn.team1.model.Course;
+import com.revlearn.team1.repository.CourseRepo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CourseService {
+    CourseRepo courseRepo;
```

Comment:
my linter is telling me: CourseRepo should be 'final' to ensure it is injected via constructor and can't be changed